sdk/loadtestservice/Azure.Developer.LoadTesting/SampleCodes/sample.jmx

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
did you intend to check this in?

Reply:
I am not sure if I understood your question correctly, can you please elaborate?